translations/en.json

```diff
@@ -339,9 +339,9 @@
     "cryptokitty-name": "CryptoKitty #{{id}}",
     "network-id": "Network ID",
     "connection-problem": "Messages connection problem",
-    "contact-code": "Contact code",
+    "contact-code": "chat key",
```

Comment:
I removed the capital letters as requested by @rachelhamlin [here](https://github.com/status-im/status-react/issues/8846#issuecomment-526645032).
I can revert this, but I agreed with her in the first place that this makes it odd in the middle of sentences. 
Can you confirm your request ?

Reply:
I think @rachelhamlin meant inside the sentences. I'll ask her again, thanks!

Reply:
Yes, thanks guys - I would write it as follows:

`Send me your chat key.`
`Chat keys are great.` :) 

Essentially, don't treat it as a proper noun. If UX disagrees on this, well - they're on vacation so we can hash it out later 😂

@churik @MajorTomSec

Reply:
btw, fwiw, I remember there being "Chat key" with key "chat-key" used in `share` pop-up in Profile tab which I worked on. Which together with this would be kind of duplicate. Translators are going to be like why do we have to translate the same phrase twice?

It is possible to have only one and handle sentence-casing/small-casing in code, but I don't know what is wise. Also, I don't know what happens with `casing` functions with different languages.